examples/material_gallery/flutter.yaml

```diff
@@ -7,6 +7,16 @@ assets:
   - assets/section_components.png
   - assets/section_patterns.png
   - assets/section_usability.png
+  - packages/flutter_gallery_assets/sun.png
+  - packages/flutter_gallery_assets/clouds-0.png
+  - packages/flutter_gallery_assets/clouds-1.png
+  - packages/flutter_gallery_assets/ray.png
+  - packages/flutter_gallery_assets/sun.png
+  - packages/flutter_gallery_assets/weathersprites.json
+  - packages/flutter_gallery_assets/weathersprites.png
+  - packages/flutter_gallery_assets/icon-sun.png
+  - packages/flutter_gallery_assets/icon-rain.png
+  - packages/flutter_gallery_assets/icon-snow.png
```

Comment:
The font uses the Open Font License, as the font is rendered out (not bundled or redistributed) no attribution should be required. However, I did paste in the license in the graphics package, just in case and because it's a nice thing to do.